examples/weather/unified_recipe/curate_era5.py

```diff
@@ -88,36 +77,36 @@ def __init__(
                 needed_variables.append(variable[0])
             else:
                 needed_variables.append(variable)
-        for variables in self.arco_era5.variables:
+        for variables in self.era5.variables:
```

Comment:
Why is `variables` plural? Does `self.era5.variables` contain list of lists? If so, the the next line (`not in` check) would not work?

Reply:
Fixed the plural. `arco_era5.variables` is a just list of variable so it works